packages/backend/src/Application.ts

```diff
@@ -200,14 +200,16 @@ export class Application {
       await database.migrateToLatest()
 
       if (config.syncEnabled) {
```

Comment:
Do we actually want/need this? Feature-specific flags seem to be sufficient @krzkaczor @sz-piotr

Reply:
IMO we should remove it, but this can be done in a separate PR.